wordpress-mobile/AztecEditor-iOS
# Line Break Support

Ref. #492

### To test:
1. Please, verify the (new) unit tests are green
2. Toggle H1 and insert `Hello\nWorld`. Verify that the HTML looks like: `<h1>Hello</h1>World`
3. Type `Hello\nWorld`. Verify that the HTML looks like: `Hello<br>World`

*Note:*
- `testNewlineDoesNotGetAddedBetweenTwoBlocklevelElements` is known to break. Working on a fix right now!
